halo2-snark-aggregator-api/src/tests/systems/halo2.rs

```diff
@@ -1,5 +1,5 @@
 pub mod add_mul_test;
 pub mod lookup_test;
 
-#[cfg(test)]
-pub mod zkevm_test;
+// #[cfg(test)]
```

Comment:
Does this break readme doc？

Reply:
fixed in https://github.com/scroll-tech/halo2-snark-aggregator/pull/35/commits/978e3e2a772adcc9ff50f3c1d999a7e6dd9829c4